src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+use Greg0ire\Enum\AbstractEnum;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator = null)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
+     * Displays the label corresponding to a specific value of an enumeration.
+     *
+     * @param mixed       $value             Must exist in the enumeration class specified with $class
+     * @param string      $class             The enum class name
+     * @param string|bool $translationDomain The translation domain to use if the translator if available.
+     *                                       string: Use the specified one
+     *                                       null: Use the default one
```

Comment:
> You're going to hate me

:gun: 

> Maybe there should be two functions, mapped to one filter by configuration?

Can you elaborate? For me, I was talking about not managing the translator at all and let the `trans` filter do his job.

> How about you do your use case and I do mine?

My case with the actual state of this PR is pretty simple:

```twig
{{ member.accessLevel|enum_label('AppBundle\\Enum\\OrganizationAccessLevel', 'global')
```

I think yours is pretty the same, with `my_model` instead of `global`. Am I right?